### PR TITLE
Add Pre-Protocols for Meetings.

### PR DESCRIPTION
**Pull request by deiferni**
This PR adds a Pre-Protocol form for Meetings, to do so we:
 - Add a form for meeting and meeting participants.
 - Add database migrations to add new columns to hold the new form data.
 - Display a link to that edit view on the meeting overview page.

We also refactor existing code by splitting the quite big module meetings into submodules in a new meetings package.

For the sake of convenience we also create example committee memberships when installing the `examplecontent` profile.

### CODE REVIEW

File: opengever/meeting/browser/meetings/templates/pre_protocol.pt [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                         Considerations
+                  </label>
+                  <textarea class="textarea-widget text-field"
+                            tal:attributes="name string:${name}.considerations:record;
+                                            id string:${name}.considerations:record"
+                            tal:content="protocol/considerations"/>
+
+                  <label tal:attributes="for string:${name}.proposed_action:record"
+                         i18n:translate="">
+                         Proposal
```
Comment (lukasgraf) [inline opengever/meeting/browser/meetings/templates/pre_protocol.pt]: `Proposal` -> `Proposed action`?

File: opengever/meeting/browser/meetings/preprotocol.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    other_participants = schema.Text(
+        title=_(u"label_other_participants", default=u"Other Participants"),
+        required=False)
+
+
+class EditPreProtocol(AutoExtensibleForm, ModelEditForm, EditForm):
+
+    ignoreContext = True
+    schema = IParticipants
+    content_type = Meeting
```
Comment (lukasgraf) [inline opengever/meeting/browser/meetings/preprotocol.py]: Is this actually correct?

**Comment by lukasgraf** [comment]
:+1:

File: opengever/meeting/browser/meetings/templates/pre_protocol.pt (hunk 1 above)
Comment (deiferni) [inline opengever/meeting/browser/meetings/templates/pre_protocol.pt]: :+1:

File: opengever/meeting/browser/meetings/preprotocol.py (hunk 2 above)
Comment (deiferni) [inline opengever/meeting/browser/meetings/preprotocol.py]: yes :wink:

**Pull request merged**